content/applications/general/users/access_rights.rst

```diff
@@ -52,6 +52,65 @@ The :guilabel:`Administration` field in the :guilabel:`Access Rights` tab has th
 .. image:: access_rights/user-permissions-dropdown-menu.png
    :alt: The Sales apps drop-down menu to set the user's level of permissions.
 
+Manage specific permissions
+---------------------------
+
+While access rights are typically assigned in bundles under specific roles, they can also be set as
+explicit permissions.
+
+.. example::
+   For example, giving a user the :guilabel:`Administrator` permission for **Timesheets**
+   gives them full access to that app. That user, while holding full access, can *still* have their
+   ability to manage *their own* timesheets restricted — such as in the case of a salaried payroll
+   administrator who does not need to track time.
+
+To manage specific permissions, :ref:`developer mode <developer-mode>` must be enabled. After that,
+navigate to the :menuselection:`Settings` app. Then click :guilabel:`Manage Users`, select a user,
+and go to the :guilabel:`Technical Access Rights` tab. From here, :guilabel:`Groups` can be edited,
+and specific access rights can be managed across the various sections. If no changes are made to
+these *Groups*, then their permissions will mirror the selections made in the :guilabel:`Access
+Rights` tab.
```

Comment:
Totally optional, I missed the dev mode step on my first pass! I recommend breaking out this instruction from the paragraph to help it stand out? 
```suggestion
To manage specific permissions, :ref:`developer mode <developer-mode>` must be enabled.

After that, navigate to the :menuselection:`Settings` app. Then click :guilabel:`Manage Users`,
select a user, and go to the :guilabel:`Technical Access Rights` tab. From here, :guilabel:`Groups`
can be edited, and specific access rights can be managed across the various sections. If no changes
are made to these groups, then their permissions will mirror the selections made in the
:guilabel:`Access Rights` tab.
```